app/models/certificate.server.ts

```diff
@@ -18,6 +18,19 @@ export async function getCertificateByUsername(username: Certificate['username']
     .then(([certificate]) => certificate);
 }
 
+export async function getCertificatePendingByUsername(username: Certificate['username']) {
```

Comment:
@Ririio 
Instead of this I'd modify `getCertificateByUsername` to get the cert with the highest `id` instead of `validFrom`.
Since `id` is an autoincrement, not null field, certificates will be added in a sequential order (each gets an id 1 larger than the last)

Reply:
With this change the logic becomes easier, you only have to do 1 sql query

Reply:
oh what the heck that's true lmfao

Reply:
Good idea.